### ISSUE TITLE
Move logging of the server start event into the nearest prev if-block.

### ISSUE THREAD

**Issue by rgolubtsov** [rgolubtsov/transroutownish-proto-bus-cowboy#22]
Please move the logging of the server start event into the nearest previous `if`-block to prevent this logging when an error occured during server start up. &mdash; This is in the `apps/bus/src/bus_controller.erl` module:
```
...
    if (element(1, Status_) =:= error) ->
        if (element(2, Status_) =:= eaddrinuse) ->
            logger:critical(?ERR_CANNOT_START_SERVER?ERR_ADDR_ALREADY_IN_USE);
           (true) ->
            logger:critical(?ERR_CANNOT_START_SERVER?ERR_SERV_UNKNOWN_REASON)
        end,

        init:stop(?EXIT_FAILURE);
       (true) -> false
    end,

    ServerPort_ = integer_to_list(ServerPort),

    logger:info(              ?MSG_SERVER_STARTED ++ ServerPort_),
    syslog:log (Syslog, info, ?MSG_SERVER_STARTED ++ ServerPort_).
...
```

**Comment by rgolubtsov** [comment]
`HTTP 202 Accepted` :+1:

**Comment by rgolubtsov** [comment]
`HTTP 200 OK` :+1:
Fixed by #23 .
